mLRS/Common/fhss.h

```diff
@@ -29,7 +29,10 @@
 #ifdef FREQUENCY_BAND_433_MHZ
 #define FHSS_HAS_CONFIG_433_MHZ
 #endif
-#ifdef FREQUENCY_BAND_868_MHZ
+#ifdef FREQUENCY_BAND_70_CM_HAM
+#define FHSS_HAS_CONFIG_70_CM_HAM
+#endif
+#if defined FREQUENCY_BAND_868_MHZ
```

Comment:
here it went a bit wrong, should have stayed` #ifdef FRE...`

Reply:
Fixed